PF-build.sh

```diff
@@ -76,32 +88,67 @@
 #                          $2 = multi language OR english only [ALL/EN_ONLY]
 #                          $3 = development status [GOLD/RC/BETA/ALPHA/DEVEL/DEBUG]
 #                          If one argument is wrong a list of valid one will be shown
-# 13 Mar 2019, 3d-gussner, MKbel updated the linux build enviromentto version 1.0.2 with an Fix maximum firmware flash size.
+# 13 Mar 2019, 3d-gussner, MKbel updated the linux build environment to version 1.0.2 with an Fix maximum firmware flash size.
 #                          So did I
+# 11 Jul 2019, deliopoulos,Updated to v1.0.6 as Prusa needs a new board definition for Firmware 3.8.x86_64
+#						   - Splitted the Download of Windows Arduino IDE 1.8.5 and Prusa specific part
+#                            --> less download volume needed and saves some time
+#
+# 13 Jul 2019, deliopoulos,Splitting of Ardunio IDE and Prusa parts also for Linux64
+# 13 Jul 2019, 3d-gussner, Added Linux 32-bit version (untested yet)
+#                          MacOS could be added in future if needs
+# 14 Jul 2019, 3d-gussner, Update preferences and make it really portable
+# 15 Jul 2019, 3d-gussner, New PF-build-env gihub branch
+# 16 Jul 2019, 3d-gussner, New Arduino_boards github fork
+# 17 Jul 2019, 3d-gussner, Final tests under Windows 10 and Linux Subsystem for Windows   
+# 18 Jul 2019, 3d-gussner, Added python check
+# 18 Jul 2019, deliopoulos, No need more for changing 'platform.txt' file as it comes with the Arduino Boards.
+# 18 Jul 2019, deliopoulos, Modified 'PF_BUILD_FILE_URL' to use 'BUILD_ENV' variable
+# 22 Jul 2019, 3d-gussner, Modiffied checks to check folder and/or installation output exists.
+# 22 Jul 2019, 3d-gussner, Added check if Arduino IDE 1.8.5 boards have been updated
+# 22 Jul 2019, 3d-gussner, Changed exit numbers 1-13 for prepare build env 21-28 for prepare compiling 31-36 compiling
+# 22 Jul 2019, 3d-gussner, Changed BOARD_URL to DRracers respository after he pulled my PR https://github.com/DRracer/Arduino_Boards/pull/1
 
-
-###Check if OSTYPE is supported
+#### Start check if OSTYPE is supported
+# Windows
 if [ $OSTYPE == "msys" ]; then
 	if [ $(uname -m) == "x86_64" ]; then
 		echo "$(tput setaf 2)Windows 64-bit found$(tput sgr0)"
+		Processor="64"
+	elif [ $(uname -m) == "i386" ]; then
+		echo "$(tput setaf 2)Windows 32-bit found$(tput sgr0)"
+		Processor="32"
 	fi
+# Linux 64-bit
 elif [ $OSTYPE == "linux-gnu" ]; then
 	if [ $(uname -m) == "x86_64" ]; then
 		echo "$(tput setaf 2)Linux 64-bit found$(tput sgr0)"
+		Processor="64"
+	fi
+# Linux 32-bit
+elif [ $OSTYPE == "linux-gnu" ]; then
+	if [ $(uname -m) == "i386" ]; then
```

Comment:
This condition is not executed on 32-bit systems with $OSTYPE == "linux-gnu" - that results to unset Processor variable and failed wget.

Reply:
Thanks for the feedback and testing on so many different OS.
Modified the if statement .. hope it detects it now better.